pkg/graveler/graveler.go

```diff
@@ -748,6 +755,9 @@ type Collaborator interface {
 
 	// ListPullRequests lists pull requests on repository
 	ListPullRequests(ctx context.Context, repository *RepositoryRecord) (PullsIterator, error)
+
+	// UpdatePullRequest delete pull request from the given repository
```

Comment:
>// UpdatePullRequest update pull request in the given repository

Reply:
Please use github suggestions next time - it'll make it easier to update

Reply:
Please use github suggestions next time - it'll make it easier to update